README.md

```diff
@@ -1096,6 +1098,23 @@ let mutation = Storefront.buildMutation { $0
 }
 ```
 
+###### Updating shipping line [⤴](#table-of-contents)
```

Comment:
I think this section should go after `Polling for shipping rates` section, as it sounds more logical first you poll for rates then apply it